src/operator/tensor/elemwise_binary_op.h

```diff
@@ -400,6 +815,20 @@ void BinaryBackwardUseInWithHalf2(const nnvm::NodeAttrs& attrs,
   .add_argument("lhs", "NDArray-or-Symbol", "first input")          \
   .add_argument("rhs", "NDArray-or-Symbol", "second input")
 
+/*! \brief Binary launch */
+#define MXNET_OPERATOR_REGISTER_BINARY_WITH_SPARSE_CPU(__name$, __kernel$)               \
+  MXNET_OPERATOR_REGISTER_BINARY(__name$)                                            \
+  .set_attr<FInferStorageType>("FInferStorageType", ElemwiseStorageType<2, 1>)       \
+  .set_attr<FCompute>("FCompute<cpu>", ElemwiseBinaryOp::Launch<cpu, __kernel$>)     \
+  .set_attr<FComputeEx>("FComputeEx<cpu>", ElemwiseBinaryOp::LaunchEx<cpu, __kernel$>)
+
+/*! \brief Binary launch, dense result */
+#define MXNET_OPERATOR_REGISTER_BINARY_WITH_SPARSE_CPU_DR(__name$, __kernel$)              \
+  MXNET_OPERATOR_REGISTER_BINARY(__name$)                                              \
+  .set_attr<FInferStorageType>("FInferStorageType", ElemwiseStorageTypeDenseOutput<1>) \
+  .set_attr<FCompute>("FCompute<cpu>", ElemwiseBinaryOp::Launch<cpu, __kernel$>)       \
```

Comment:
Rename to ElemwiseBinaryOp::Compute and ComputeEx

Reply:
I thought I changed all of those. I wonder which branch I changed them on..